WORKSPACE

```diff
@@ -35,9 +35,9 @@ bind(
 # When updating envoy sha manually please update the sha in istio.deps file also
```

Comment:
Could you add the information here:
```
# envoy commit date 05/15/2019
# bazel version: 0.25.0
```
It was always a pain to find the correct bazel version to build

Reply:
done